### ISSUE TITLE
Explain Your Issue

### ISSUE THREAD

**Issue by marcovitali** [wolveix/satisfactory-server#210]
Hi,
After the promotion of Update 8 on the Early Access branch, it is impossible to access on the dedicated server. From the client the server appears to be offline.

Reading online it seems that the problem is due to the fact that the server now listens on IPv6 by default. It seems to be solved by passing the parameter `-multihome=0.0.0.0` to the startup command.

`./FactoryServer.sh "-multihome=0.0.0.0"`

Would it be possible to investigate?

Thank you very much.

**Comment by wolveix** [comment]
@marcovitali see if the issue persists when setting `SERVERIP=0.0.0.0`

**Comment by marcovitali** [comment]
> @marcovitali see if the issue persists when setting `SERVERIP=0.0.0.0`.
> 
> If it does, `-multihome` has no effect. Please provide your full runtime command and system information (as the issue template suggests). Thanks!

Yes, I had already tried that environment variable, but it still didn't work.
I simply use your Docker image via Docker Compose.

```dockerfile
satisfactory-server:
  image: 'wolveix/satisfactory-server:latest'
  container_name: 'satisfactory-server'
  ports:
    - '7777:7777/udp'
    - '15000:15000/udp'
    - '15777:15777/udp'
  restart: 'unless-stopped'
  environment:
    AUTOSAVENUM: 10
    CRASHREPORT: 'false'
    MAXTICKRATE: 30
    SERVERIP: '0.0.0.0'
    TIMEOUT: 30
  volumes:
    - '${APPDATA_PATH}/satisfactory-server:/config'
```

**Comment by wolveix** [comment]
Please provide your system info as well. What log does the server output when running?

**Comment by wolveix** [comment]
Per the issue template:

> **System Specs (please complete the following information):**
> If you're on Linux, just paste the following block as a single command, and paste the output here.
> ```
> echo "===== START ISSUE REPORT =====
> OS:  $(uname -a)
> CPU: $(lscpu | grep 'Model name:' | sed 's/Model name:[[:space:]]*//g')
> RAM: $(awk '/MemAvailable/ {printf( "%d\n", $2 / 1024000 )}' /proc/meminfo)GB/$(awk '/MemTotal/ {printf( "%d\n", $2 / 1024000 )}' /proc/meminfo)GB
> HDD: $(df -h | awk '$NF=="/"{printf "%dGB/%dGB (%s used)\n", $3,$2,$5}')
> ===== END ISSUE REPORT ====="
> ```
> 
> Alternatively, you can find the information manually. Here's what we're looking for:
> -   OS: [e.g. Ubuntu 18.04 x86_64] (Linux: `uname -a`)
> -   CPU: [e.g. AMD Ryzen 5 3600 6-Core Processor] (Linux: `lscpu`)
> -   RAM: [e.g. 4GB/16GB] (Linux: `cat /proc/meminfo | grep Mem`)
> -   HDD; [e.g. 22GB/251GB (9% used)] (Linux: `df -h`)

**Comment by ElijahTailor** [comment]
Encountering the same issue.

**Comment by wolveix** [comment]
@ElijahTailor could you please provide the same info? Docker runtime, system specs, and runtime log :)

**Comment by ElijahTailor** [comment]
> @ElijahTailor could you please provide the same info? Docker runtime, system specs, and runtime log :)

```
===== START ISSUE REPORT =====
OS:  Linux mp-srv 5.15.0-86-generic #96-Ubuntu SMP Wed Sep 20 08:23:49 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
CPU: Intel Xeon Processor (Skylake, IBRS)
RAM: 21GB/22GB
HDD: 32GB/94GB (35% used)

===== END ISSUE REPORT =====
```

Can't get what do you mean by Docker runtime. Latest log attached.
[sf-log.txt](https://github.com/wolveix/satisfactory-server/files/13370338/sf-log.txt)

**Comment by wolveix** [comment]
@ElijahTailor that looks like a very old error. Try running `docker compose pull && docker compose up -d`. This will update your downloaded version of the Docker image.

If you're not using Docker Compose, do `docker pull wolveix/satisfactory-server:latest`

**Comment by marcovitali** [comment]
Sorry @wolveix, I don't know for what reason, maybe some coincidence, but by removing and re-adding the server within the client everything went back to working properly.
Thank you for your support.

**Comment by wolveix** [comment]
@marcovitali no problem, I'm glad you got it resolved!
